cmd/kube-apiserver/app/testing/testserver.go

```diff
@@ -214,11 +214,13 @@ func StartTestServer(t Logger, instanceOptions *TestServerInstanceOptions, custo
 
 	s.ServiceClusterIPRanges = "10.0.0.0/16"
 	s.Etcd.StorageConfig = *storageConfig
-	s.APIEnablement.RuntimeConfig.Set("api/all=true")
```

Comment:
this was not overridden by the flags, so it has always all the apis enabled, at least this was always present on the RuntimeConfig

Reply:
`api/all` is not the default, this enables all alpha and beta APIs as well